lib/src/widgets/movie_detail_subtitle.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import '/../utils/movie_colors.dart';
+import '/../utils/movie_dimensions.dart';
+
+class MovieDetailSubtitle extends StatelessWidget {
+  final String movieReleaseDate;
+  final String movieOriginalLanguage;
+  final int movieVoteCount;
+
+  const MovieDetailSubtitle({
+    Key? key,
+    required this.movieReleaseDate,
+    required this.movieOriginalLanguage,
+    required this.movieVoteCount,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      "${this.movieReleaseDate}, ${this.movieOriginalLanguage}, ${this.movieVoteCount} votes",
```

Comment:
"votes" can be added to movieVoteCount when you create MovieDetailSubtitle
So when you create the widget you can do something like this:
MovieDetailSubtitle(
      movieReleaseDate: snapshot.data[x].movieReleaseDate,
      movieOriginalLanguage: snapshot.data[x].originalLanguage,
      movieVoteCount: snapshot.data[x].voteCount.toString() + "votes",
)
x is some position of array

Reply:
Another approach if you only use voteCount as String, change parameter value to String, and transform after you pass it as parameter of Widget

Reply:
You can now find "votes" as a parameter when the widget is created and removed from the widget file itself. Fixed.